pandas/io/json/_json.py

```diff
@@ -302,7 +302,9 @@ def __init__(
 
         # NotImplemented on a column MultiIndex
         if obj.ndim == 2 and isinstance(obj.columns, MultiIndex):
-            raise NotImplementedError("orient='table' is not supported for MultiIndex")
+            raise NotImplementedError(
```

Comment:
Did this need to change or did black format this?

Reply:
Orient=table supports MultiIndex Index but doesn't support MultiIndex on columns, I thought it was worth clarifying.